structure/base/base.go

```diff
@@ -116,3 +120,20 @@ func (b *Base) WithReference(reference string) *Base {
 	}
 	return base
 }
+
+func (b *Base) WithReferenceNoWarning(reference string) *Base {
```

Comment:
This function seems to be unused. Was it really needed ?

Reply:
correct